extract-docker-tag/action.yml

```diff
@@ -11,6 +11,9 @@ outputs:
   tag:
     description: 'Extracted Docker image tag, e.g. `pr-add-features`'
     value: ${{ steps.extract.outputs.tag }}
+  version:
+    description: 'Extracted Docker image tag by version, e.g. `0.1.0`'
+    value: ${{ steps.extract.outputs.version }}
```

Comment:
No, we should use the same `tag` output variable above. For Docker images `ferretdb/ferretdb:latest` and `ferretdb/ferretdb:v0.1.0` the tag value should be `latest` and `v0.1.0`.

Reply:
version removed. 
also refactored as agreed on the call.